server/notebooks/serializers.py

```diff
@@ -6,8 +6,7 @@
 
 class NotebookLatestRevisionField(serializers.RelatedField):
     def get_attribute(self, obj):
-        return NotebookRevision.objects.filter(notebook_id=obj.id).order_by(
-            '-created').first()
+        return NotebookRevision.objects.filter(notebook_id=obj.id).last()
```

Comment:
We can rely on .last() providing the latest revision for a notebook because we define a sorting order in the notebook model (thanks to @jaredkerim for that trick):

https://github.com/iodide-project/iodide/blob/31bdd4022306c996a63829d76aacf6c72a76b69c/server/notebooks/models.py#L50